src/vcpkg/commands.applocal.cpp

```diff
@@ -0,0 +1,550 @@
+            }
+            else if (target_binary_name == "Qt5Multimediad.dll" || target_binary_name == "Qt5Multimedia.dll")
+            {
+                deployPluginsQt("audio", target_binary_dir, qt_plugins_dir);
+                deployPluginsQt("mediaservice", target_binary_dir, qt_plugins_dir);
+                deployPluginsQt("playlistformats", target_binary_dir, qt_plugins_dir);
+            }
+            else if (target_binary_name == "Qt5PrintSupportd.dll" || target_binary_name == "Qt5PrintSupport.dll")
+            {
+                deployPluginsQt("printsupport", target_binary_dir, qt_plugins_dir);
+            }
+            else if (target_binary_name == "Qt5Qmld.dll" || target_binary_name == "Qt5Qml.dll")
+            {
+                std::error_code ec;
+                if (!m_fs.exists(target_binary_dir / "qml", ec))
+                {
+                    if (m_fs.exists(bin_dir / "../qml", ec))
+                    {
+                        m_fs.copy_regular_recursive(bin_dir / "../qml", target_binary_dir, VCPKG_LINE_INFO);
+                    }
+                    else if (m_fs.exists(bin_dir / "../../qml", ec))
+                    {
+                        m_fs.copy_regular_recursive(bin_dir / "../../qml", target_binary_dir, VCPKG_LINE_INFO);
+                    }
+                    else
+                    {
+                        Checks::exit_with_message(VCPKG_LINE_INFO, "FAILED", ec.message());
+                    }
+                }
+            }
+            else if (target_binary_name == "Qt5Quickd.dll" || target_binary_name == "Qt5Quick.dll")
+            {
+                std::vector<std::string> libs = {"Qt5QuickControls2.dll",
+                                                 "Qt5QuickControls2d.dll",
+                                                 "Qt5QuickShapes.dll",
+                                                 "Qt5QuickShapesd.dll",
+                                                 "Qt5QuickTemplates2.dll",
+                                                 "Qt5QuickTemplates2d.dll",
+                                                 "Qt5QmlWorkerScript.dll",
+                                                 "Qt5QmlWorkerScriptd.dll",
+                                                 "Qt5QuickParticles.dll",
+                                                 "Qt5QuickParticlesd.dll",
+                                                 "Qt5QuickWidgets.dll",
+                                                 "Qt5QuickWidgetsd.dll"};
+                for (std::string lib : libs)
+                {
+                    deploy_binary(target_binary_dir, bin_dir, lib);
+                }
+
+                deployPluginsQt("scenegraph", target_binary_dir, qt_plugins_dir);
+                deployPluginsQt("qmltooling", target_binary_dir, qt_plugins_dir);
+            }
+
+            std::error_code ec;
+            std::vector<Path> children = m_fs.get_files_non_recursive(target_binary_name, ec);
```

Comment:
What does this line do?
1. Isn't `target_binary_name` a filename -- not a relative or absolute path to a directory? What files should this find?
2. Why is this so structurally different than the lines in qtdeploy.ps1? https://github.com/microsoft/vcpkg/blob/8d9c52191e039fe49c580807efee502d22f32786/ports/qt5-base/qtdeploy.ps1#L95